deployment/1_setup_test_environment.js

```diff
@@ -65,7 +66,13 @@ const deploy = async (network, secret) => {
         // Deploy Kyber Network if needed
         const address = await deployKyber(deployer);
         configurator.updateKyberContract(address);
-    }
+	}
+	
+	if (config.defi.maker.deployOwn) {
+        // Deploy Kyber Network if needed
```

Comment:
Comment should be "Deploy Maker's mock Migration contract if needed"

Reply:
done